### PR TITLE
fix: [master] Fixing problems with go run in Go 1.16+

### LINKED ISSUE #504: [BUG] go run github.com/fabric8-analytics/cli-tools/gomanifest doesn't work, but gomanifest itself does, and the extension is trying for go run ...
**Describe the bug**
Im not sure but I think it is that the extension doesn't work very well with golang 1.16, because of the following error I'm getting when trying to get the analysis for a golang project.

This is the ouput of the "Dependency Analytics" console:
```
 CMD :go get -u github.com/fabric8-analytics/cli-tools/gomanifest && go run github.com/fabric8-analytics/cli-tools/gomanifest "/home/user/testingProject/" "/home/user/testingProject/target/golist.json" "go"
 STDOUT :  
 STDERR : no required module provides package github.com/fabric8-analytics/cli-tools/gomanifest: go.mod file not found in current directory or any parent directory; see 'go help modules'
```

So the specific error seems to be that the command: go run github.com/fabric8-analytics/cli-tools/gomanifest is not runnable, since if I run the same command from an external terminal in a different directory it gives the same error, even when the command "gomanifest" alone works (meaning "gomanifest" is installed and found in the path).

**To Reproduce**
1. Ubuntu 20.04 and VScode downloaded and installed manually.
2. Just open a golang project, and from the go.mod file open, click on the 'Open Vulnerability Report' icon:
image
3. Te following notification appears, just click on "Show Output Log"
image

**VSCode(please complete the following information):**
 - OS: Ubuntu 20.04
 - VSCode version 1.54.3
 - Dependency Analytics Version 0.3.0

### PR DESCRIPTION
**Pull request by Ortega-Dan**
Fixes #504

### CODE REVIEW

**Comment by Ortega-Dan** [comment]
One of the automated checks is saying: 
Error: No release type found in pull request title "Fixing problems with go run in Go 1.16+". Add a prefix to indicate what kind of release this pull request corresponds to (see https://www.conventionalcommits.org/).
Can I please get some help to know what it is ?

File: src/ProjectDataProvider.ts [hunk 1]
```diff
@@ -330,11 +330,11 @@ export module ProjectDataProvider {
         `-u`,
         `github.com/fabric8-analytics/cli-tools/gomanifest`,
         `&&`,
-        Config.getGoExecutable(),
-        `run`,
-        `github.com/fabric8-analytics/cli-tools/gomanifest`,
-        `"${vscodeRootpath}"`,
-        `"${goGraphFilePath}"`,
+        // running gomanifest directly for Go 1.16 support
+        `gomanifest`,
```
Comment (dgpatelgit) [inline src/ProjectDataProvider.ts:334]: This does not work on Mac OS, I think system will not find `gomanifest` if GOROOT or GOPATH is not set properly. Need to have generic command that can work for all use-case.

Comment (Ortega-Dan) [reply src/ProjectDataProvider.ts:334]: Thanks for checking on mac

Comment (dgpatelgit) [reply src/ProjectDataProvider.ts:334]: I have made generic changes to set GOPATH to temporary directory and use `gomanifest` with absolute path to temp directory. This changes are merged to master code. Can you please check if this resolves the issue your were facing? You can take latest extension VSIX from https://github.com/fabric8-analytics/fabric8-analytics-vscode-extension/suites/2525745446/artifacts/54726285

Comment (Ortega-Dan) [reply src/ProjectDataProvider.ts]: Thank you very much. It worked perfectly on Linux, but it breaks on Windows. I then closed this PR and made PR #510 to fix it for Windows.

**Pull request closed without merging**